content/en/docs/tasks/configure-pod-container/configure-pod-initialization.md

```diff
@@ -80,6 +80,11 @@ The output shows that nginx is serving the web page that was written by the init
       ...
 
 
+Exit the shell.
+
+Delete the Pod that you created for this exercise:
+
+    kubectl delete pod init-demo
```

Comment:
```suggestion
    kubectl delete -f https://k8s.io/examples/pods/init-containers.yaml
```
I'd recommend using this command to ensure that all items included in the `init-containers.yaml` are cleaned up.

Reply:
i agree with @onlydole suggestion.

Reply:
hi @onlydole . How can we get this merged?